src/Http/Response.php

```diff
@@ -22,8 +22,6 @@
  *
  * It is intended to reduce our coupling to any particular PSR15 implementation.
  */
-class Response 
-    extends \Zend\Diactoros\Response 
-    implements \Psr\Http\Message\ResponseInterface
+class Response extends \Zend\Diactoros\Response implements \Psr\Http\Message\ResponseInterface
```

Comment:
```suggestion
class Response extends \Zend\Diactoros\Response implements
    \Psr\Http\Message\ResponseInterface
```

This line is too long and hurts my eyes, which is easy to fix implements may be split across multiple lines